lib/galaxy/datatypes/constructive_solid_geometry.py

```diff
@@ -110,7 +111,7 @@ def display_peek(self, dataset):
 
 class PlyAscii(Ply, data.Text):
 
-    file_ext = "plyascii"
+    file_ext = "ply"
```

Comment:
Yes, this and the following changes are wrongly assigning the same value to the `file_ext` of different classes: this attribute should be unique and equal to what is used as `extension` in `config/datatypes_conf.xml.sample` .